service/resource/cloudformation/create.go

```diff
@@ -1,7 +1,63 @@
 package cloudformation
 
-import "context"
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awscloudformation "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/aws-operator/service/key"
+	"github.com/giantswarm/microerror"
+)
 
 func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
+	stackInput, ok := createChange.(awscloudformation.CreateStackInput)
+	if !ok {
+		return microerror.Mask(fmt.Errorf("unexpected type, expecting %T, got %T", stackInput, createChange))
+	}
```

Comment:
In other operators we have helper methods ala `toWhatever(interface) (specific.Type, error)`. It makes the calls within here way nicer because the error handling and type assertion magic is done separately. I would like to keep that style.

Reply:
ok, added `toCreateStackInput`, i've taken also the opportunity to add `toStackState`